src/functions.test.js

```diff
@@ -0,0 +1,43 @@
+const functions = require('./functions');
+
+test('return capitalize string', () => {
+  expect(functions.capitalizeString('string')).toEqual('String');
+});
+
+test('return reversed string', () => {
+  expect(functions.reverseString('string')).toEqual('gnirts');
+});
+
+test('return the sum between 2 and 2', () => {
+  expect(functions.calculator(2, 2).add()).toEqual(4);
+});
+
+test('return the substraction between 2 and 2', () => {
+  expect(functions.calculator(2, 2).subtract()).toEqual(0);
+});
+
+test('return the division between 2 and 2', () => {
+  expect(functions.calculator(2, 2).divide()).toEqual(1);
+});
+
+test('return the multiplication between 2 and 2', () => {
+  expect(functions.calculator(2, 2).multiply()).toEqual(4);
+});
+
+test('using punctuation marks', () => {
+  expect(functions.cipher('helló world.', 1).ceasar()).toEqual('ifmmó xpsme.');
+});
+
+test('return encrypted alphabet', () => {
+  expect(functions.cipher('abcdefghijklmnopqrstuvwxyz', 1).ceasar()).toEqual('bcdefghijklmnopqrstuvwxyza');
+});
+
+test('key is greater than alphabet length', () => {
+  expect(functions.cipher('helló world.', 10).ceasar()).toEqual('rovvó gybvn.');
+});
```

Comment:
You missed a test that checked for keeping the same case and in so doing missed that your function does not handle this well and gives wrong results. 
For example, passing `functions.cipher('Z!', 1).ceasar()` should return `A!` but instead return `Z!` (Not that it simply returns original values of uppercase letters)
Please add the test and fix the original object.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/caesar_requirement.png) from Odin description.

Reply:
It is true, now it is working.